userpanel/modules/finances/style/bclean/templates/userbalancebox.html

```diff
@@ -58,31 +56,31 @@
 					{section name=balancelist loop=$balancelist.list step=-1}
 						{$item=$balancelist.list[balancelist]}
 						{if $item.time > $ldate}
-						<tr class="table-click {if $item.value > 0}table-success{else}table-danger{/if}" {if $item.doctype == $smarty.const.DOC_INVOICE || $item.doctype == $smarty.const.DOC_CNOTE}data-href="?m=finances&f=invoice&id={$item.docid}"{/if}>
-							<td class="text-right text-nowrap">{$item.date|date_format:"%Y-%m-%d"}</td>
+						<tr class="table-click {if $chilek_modeon}{if $item.value > 0}table-success{else}table-danger{/if}{/if}" {if $item.doctype == $smarty.const.DOC_INVOICE || $item.doctype == $smarty.const.DOC_CNOTE}data-href="?m=finances&f=invoice&id={$item.docid}"{/if}>
```

Comment:
A to co? :rofl: ?

Reply:
Likwidacja dziewnych kolorków na które się uparłeś ;-D

Reply:
Otwórz dwa okienka i porównaj.

Reply:
W samej kolumnie balans - wystarczy.

Reply:
Ja nie jestem przeciwny ogólnie obecności kolorów ale te obecne są źle dobrane.

Reply:
Ten PR ujednolica klasy tabel trochę bo obecnie w origin/master każda jest z innej parafii.